# Fix a FTBFS with system libnss (caused by bad linking order with libcrmf)
# by Fabien Tassin <fta@sofaraway.org>

Index: seamonkey-1.1.4/configure.in
===================================================================
--- seamonkey-1.1.4.orig/configure.in
+++ seamonkey-1.1.4/configure.in
@@ -3735,17 +3735,17 @@
 [  --with-system-nss      Use system installed NSS],
     _USE_SYSTEM_NSS=1 )
 
 if test -n "$_USE_SYSTEM_NSS"; then
     AM_PATH_NSS(3.0.0, [MOZ_NATIVE_NSS=1], [MOZ_NATIVE_NSS=])
 fi
 
 if test -n "$MOZ_NATIVE_NSS"; then
-   NSS_LIBS="$NSS_LIBS -lcrmf"
+   NSS_LIBS=" -lcrmf $NSS_LIBS"
 else
    NSS_CFLAGS='-I$(DIST)/public/nss'
    NSS_DEP_LIBS='\\\
         $(DIST)/lib/$(LIB_PREFIX)crmf.$(LIB_SUFFIX) \\\
         $(DIST)/lib/$(DLL_PREFIX)smime'$NSS_VERSION'$(DLL_SUFFIX) \\\
         $(DIST)/lib/$(DLL_PREFIX)ssl'$NSS_VERSION'$(DLL_SUFFIX) \\\
         $(DIST)/lib/$(DLL_PREFIX)nss'$NSS_VERSION'$(DLL_SUFFIX) \\\
         $(DIST)/lib/$(DLL_PREFIX)softokn'$NSS_VERSION'$(DLL_SUFFIX)'

